java/client/src/org/openqa/selenium/chrome/ChromeDriverInfo.java

```diff
@@ -41,7 +43,7 @@ public String getDisplayName() {
 
   @Override
   public Capabilities getCanonicalCapabilities() {
-    return new ImmutableCapabilities(CapabilityType.BROWSER_NAME, BrowserType.CHROME);
+    return new ImmutableCapabilities(CapabilityType.BROWSER_NAME, BrowserType.CHROME, PLATFORM_NAME, System.getProperty("os.name"));
```

Comment:
This isn't correct. It means that only the current platform can be used for a remote webdriver.